### PR TITLE
v0.1.0 closes #265

### LINKED ISSUE #265: v0.1.0

### PR DESCRIPTION
**Pull request by bigdatasourav**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+## v0.1.0 [2021-08-04]
+
+_What's new?_
+
+_Bug fixes_
+
+- Fixed: Restrict get api call only to root compartment and one zone per region ([#264](https://github.com/turbot/steampipe-plugin-oci/pull/264))
```
Comment (cbruno10) [inline CHANGELOG.md:7]:
```suggestion
- Fixed: Restrict get API calls to the root compartment and one zone per region in several core and file storage tables ([#264](https://github.com/turbot/steampipe-plugin-oci/pull/264))
```

We can add a bit more information here to let users know what types of tables are affected. Also `API` should be capitalized.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
+## v0.1.0 [2021-08-04]
```
Comment (cbruno10) [inline CHANGELOG.md:1]: When releasing, please remember to update this date

**Review by cbruno10 — CHANGES_REQUESTED** [review]
Please see comments, thanks!

**Review by rajlearner17 — APPROVED** [review]
LGTM

**Review by cbruno10 — APPROVED** [review]

**Pull request merged**